File: Python/tests/test_bandit2arm_delta.py
import pytest

from hbayesdm.models import bandit2arm_delta


def test_bandit2arm_delta():
    _ = bandit2arm_delta(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bandit2arm_delta.R
context("Test bandit2arm_delta")
library(hBayesDM)

test_that("Test bandit2arm_delta", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bandit2arm_delta(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
